contributing.md

```diff
@@ -1,14 +1,70 @@
-#### How to reach us
-The mailing list for developers : systemroles@lists.fedorahosted.org
-
-[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
-
-[Archive of the mailing list](https://lists.fedorahosted.org/archives/list/systemroles@lists.fedorahosted.org/)
-
-If you are using IRC, join the `#systemroles` IRC channel on freenode
-
-
-#### Write a good commit message
+# Contributing to the Network Linux System Role
+
+## Where to start
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/network/issus). Issues labeled
+with [**help wanted**](https://github.com/linux-system-roles/network/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) are likely to be suitable for new contributors!
+- **Code** is managed on
+  [Github](https://github.com/linux-system-roles/network), using [Pull
+Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
+
+## How to contribute
+
+1. Make a
+   [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+of this repository.
+
+2. Create a new git branch on your local fork (the name is not relevant) and
+   make the changes you need to complete an issue.
+
+3. Do not forget to run unit and integration tests before pushing any changes!
+  
+  - This project uses [tox](https://tox.readthedocs.io/en/latest/) to run unit
+    tests. You can try it with `tox -e py36` in case you want to try it using
+    Python 3.6, or just `tox` if you want to run all the tests.
+
+  - Format the code using [Python black](https://black.readthedocs.io/en/stable/)
+
+  - Check the YML files are correctly formatted using `tox -e yamllint`.
+
+  - Integration tests are executed as
+    [ansible-playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks.html).
+To run them you can use this [default CentOS
+VM](https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2)
+and execute the command (note that the last path is the one of the test you
+want to run):
+  
+``TEST_SUBJECTS=CentOS-8-GenericCloud-8.1.1911-20200113.3.x86_64.qcow2
+ansible-playbook -v -i /usr/share/ansible/inventory/standard-inventory-qcow2
+tests/test_default.yml``
```

Comment:
This should probably also mention that the inventory script is part of the standard-test-roles-inventory-qemu package.